pyaedt/edb_core/padstack.py

```diff
@@ -529,6 +529,32 @@ def set_all_antipad_value(self, value):
                 padstack.edb_padstack.SetData(cloned_padstack_data)
             return all_succeed
 
+    @pyaedt_function_handler
+    def check_and_fix_via_plating(self, minimum_value_to_replace=0.0, default_plating_ratio=0.2):
+        """Check and fix zero value plating ratio via definition with assigning default value.
```

Comment:
I'm not sure I understand this description. Can it be shortened. Maybe something like this--and if it's wrong, it shows how I truly don't comprehend what it says:

'''Check and fix the zero value plating ratio.

This method allows you to assign a default value using a definition.

Reply:
Yes your are right it's not clear. I will change it